src/main/java/com/school/schedule/app/Parent.java

```diff
@@ -0,0 +1,39 @@
+package com.school.schedule.app;
+
+import java.util.List;
+
+public class Parent extends Person{
+
+    private List<Student> children;
```

Comment:
It may be more intuitive to have the `Student` class point to their `Parent` objects. Typically a school would like to quickly find the parents of a specific student.

Reply:
Changed that as well !